src/app/ubs/ubs-admin/components/ubs-admin-customers/ubs-admin-customer-details/ubs-admin-customer-details.component.ts

```diff
@@ -20,4 +32,31 @@ export class UbsAdminCustomerDetailsComponent implements OnInit {
     this.localStorageService.removeCurrentCustomer();
     this.location.back();
   }
+
+  openPopUp(column: string, data: string, userId: string): void {
```

Comment:
There are potential issues, including mismanagement of asynchronous translation subscriptions  and unguarded operations:
```
openPopUp(column: string, chatLink: string | null, userId: string | null): void{
if (!userId || !chatLink) {}
}
```